app/tests/unit/components/Form/ProjectManagerForm.test.tsx

```diff
@@ -0,0 +1,191 @@
+                  rowId: 2,
+                  label: "Test Label 2",
+                },
+                formChange: {
+                  id: "Change 2 ID",
+                  newFormData: {
+                    cifUserId: 2,
+                    projectId: 1,
+                    projectManagerLabelId: 2,
+                  },
+                },
+              },
+            },
+          ],
+        },
+        projectFormChange: {
+          formDataRecordId: 1,
+        },
+      },
+      allCifUsers: {
+        edges: [
+          {
+            node: {
+              rowId: 1,
+              firstName: "Test First Name 1",
+              lastName: "Test Last Name 1",
+            },
+          },
+          {
+            node: {
+              rowId: 2,
+              firstName: "Test First Name 2",
+              lastName: "Test Last Name 2",
+            },
+          },
+        ],
+      },
+    };
+  },
+});
+
+describe("The ProjectManagerForm", () => {
+  beforeEach(() => {
+    jest.restoreAllMocks();
+
+    environment = createMockEnvironment();
+
+    environment.mock.queueOperationResolver((operation) =>
+      MockPayloadGenerator.generate(operation, getMockQueryPayload())
+    );
+
+    environment.mock.queuePendingOperation(compiledProjectManagerFormQuery, {});
+  });
+
+  it("Renders a form for each Project Manager Label", () => {
+    renderProjectForm();
+    expect(screen.getAllByRole("textbox")).toHaveLength(2);
+  });
+
+  it("Renders any data contained in a formChange", () => {
+    renderProjectForm();
+    expect(
+      screen.getAllByPlaceholderText("Select a Project Manager")[1]
+    ).toHaveValue("Test First Name 2 Test Last Name 2");
+  });
+
+  it("Calls the addManagerToRevision mutation when a new selection is made in the Manager dropdown", () => {
+    const mutationSpy = jest.fn();
+    jest
+      .spyOn(require("react-relay"), "useMutation")
+      .mockImplementation(() => [mutationSpy, jest.fn()]);
+
+    renderProjectForm();
+
+    fireEvent.click(screen.getAllByTitle("Open")[0]);
+    fireEvent.click(screen.getByText("Test First Name 1 Test Last Name 1"));
+
+    expect(mutationSpy).toHaveBeenCalledWith({
+      variables: {
+        projectRevision: "Test Revision ID",
+        projectRevisionId: 1,
+        newFormData: { cifUserId: 1, projectId: 1, projectManagerLabelId: 1 },
+      },
+    });
+  });
+
+  it("Calls the deleteFormChange mutation when the remove button is clicked", () => {
+    const mutationSpy = jest.fn();
+    jest
+      .spyOn(require("react-relay"), "useMutation")
+      .mockImplementation(() => [mutationSpy, jest.fn()]);
+
+    renderProjectForm();
+    const clearButton = screen.getAllByText("Clear")[1];
+    clearButton.click();
+
+    expect(mutationSpy).toHaveBeenCalledWith({
+      variables: {
+        input: {
+          id: "Change 2 ID",
+        },
+        projectRevision: "Test Revision ID",
+      },
+    });
+  });
+
```

Comment:
We're missing a test that makes sure the `updateFormChangeMutation` is called when we update a dropdown field without clearing it